guide/getting-started/why-bitcoin-is-unique.md

```diff
@@ -67,15 +67,17 @@ In addition, six fundamental properties define whether or not a currency can act
 5. Divisibility
 6. Recognizability
 
-Although bitcoin is purely digital, it meets every classical definition of what makes something money. Instead of relying on physical properties (like gold and silver) or central authorities (like government-issued fiat currencies), bitcoin relies on the world’s most powerful computer network to mathematically enforce the rules that make it the first truly digital form of cash.
+Although bitcoin is purely digital, it meets every classical definition of what makes something money. Instead of relying on physical properties (like gold and silver) or central authorities (like government-issued fiat currencies), bitcoin relies on the world’s most powerful [computer network]({{ '/guide/getting-started/technology-primer/#what-is-a-node' | relative_url }}) to mathematically enforce the rules that make it the first truly digital form of cash.
+
+Natively digital money can also be extended in previously unthinkable ways. From smart contracts to Layer 2 solutions like the Lightning Network, Bitcoin can get built upon to enable new and evolving use cases.
```

Comment:
```suggestion
Natively digital money can also be extended in previously unthinkable ways. From smart contracts to Layer 2 solutions like the Lightning Network that enable faster and cheaper payments, Bitcoin can get built upon to enable new and evolving use cases.
```

Reply:
Suggest we hint at what this is good for as the terms (Layer 2, Lightning network) aren't self explanatory.